tests/validation/tests/v3_api/test_registry.py

```diff
@@ -201,17 +479,19 @@ def create_validate_workload(p_client, ns):
     workload = p_client.reload(workload)
 
     validate_workload(p_client, workload, "deployment", ns.name)
+    return workload
 
 
 def create_validate_workload_with_invalid_registry(p_client, ns):
 
-    workload = create_workload(p_client, ns)
+    workload = create_workload(p_client, ns, invalid_wl=True)
     workload = p_client.reload(workload)
+    print(workload)
     print("Workload State " + workload.state)
     # Verify that the workload fails to reach active state after the registry
     # update has been made with invalid password
-    if workload.state != "active":
-        assert True
+    assert workload.state != "active", "Invalid workload came up active!"
+    return workload
```

Comment:
Can we delete workload as part of this create_validate_workload_with_invalid_registry method so that it doesn't have to be deleted for every test case that uses this method? Remove the deletes from each test case that uses this method.

Reply:
Done